docs/getting-started.md

```diff
@@ -59,9 +59,15 @@ curl localhost:10101/index/repository -X POST
 
 Let's create the `stargazer` frame which has user IDs of stargazers as its rows:
 ```
-curl localhost:10101/index/repository/frame/stargazer -X POST
+curl localhost:10101/index/repository/frame/stargazer \
+     -X POST \
+     -d '{"options": {"timeQuantum": "YMD"}}'
 ```
 
+Since our data contains time stamps for the time users starred repos, we set the *time quantum* for the `stargazer` frame in the options as well. Time quantum is the resolution of the time we want to use, and we set it to `YMD` (year, month, day) for `stargazer`.
+
+We set `inverseEnabled` to `true` in order to allow queries over columns as well as rows.
```

Comment:
can you remove this `inverseEnabled` instruction?

Reply:
Yeah, sorry. I've just did that.